### DISCUSSION TITLE
Qiita Blog の記事内の画像が小さくて見えないので工夫してほしい

### DISCUSSION THREAD

**Discussion by k1350** [increments/qiita-discussions#182] [要望・アイデア・不具合]
[記事カードのUI変更をベータ版として公開します](https://blog.qiita.com/update-article-card-beta/) の投稿を見て、画像が小さくぼやけていて変更点がよく見えないと思いました。

Qiita は通常の投稿記事やリリースノート内の画像はクリックすると別タブで開くのですが、Qiita Blog は開きません。  
Qiita Blog 内の画像もクリックしたら別タブで開いてほしいです。

もしくは変更前後を横並びではなく縦並びにするなど、きちんと変更点が目視できるように工夫してほしいです。  

またこの記事に限って言えば、画像を右クリックして別タブで開いても依然として十分な大きさの画像ではないと思いました。  
リリースノートのように十分な大きさの画像を載せてほしいです。

**Comment by kskwtnk** [comment]
@k1350 
ご意見ありがとうございます！

申し訳ありません！キャプチャ画像が小さくなっていたため、大きいものに差し替えました 🙏
 
別タブで開くについては、Qiita BlogをWordPressから別の仕組みに置き換える話も出ており、その中で一緒に対応とさせていただけると幸いです 🙇
